### PR TITLE
 Add Two

### PR DESCRIPTION
**Pull request by Divya-tri**

### CODE REVIEW

**Comment by Vdisha-245** [comment]
Good

**Comment by Vdisha-245** [comment]
Good job

**Comment by Lead999** [comment]
Good job

**Comment by Lead999** [comment]
Good

**Review by Lead999 — COMMENTED** [review]
Very nice

**Review by Vdisha-245 — COMMENTED** [review]
Good

**Review by Anushka0123 — COMMENTED** [review]
It was great

File: Add two [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include<stdio.h>
+int main()
+{
+  int no1, no2;
+  printf("Insert two numbers:");
+  scanf("%d %d",&no1, &no2);
+
```
Comment (Vdisha-245) [inline Add two:7]: Give more than two numbers

Comment (Vdisha-245) [reply Add two:7]: Got it

File: Add two [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+int main()
+{
+  int no1, no2;
+  printf("Insert two numbers:");
+  scanf("%d %d",&no1, &no2);
+      
+  //Condition to check which of the two number is greater
+  //it will compare of number where number 1 is greater
+  if(no1 > no2)
+     printf("%d is greatest",no1);
```
Comment (Vdisha-245) [inline Add two:11]: Great

**Pull request merged**